Robust.Shared/Physics/Dynamics/Contacts/Contact.cs

```diff
@@ -417,6 +417,17 @@ public EntityUid OtherEnt(EntityUid uid)
 
             throw new InvalidOperationException();
         }
+
+        [Pure, PublicAPI]
+        public PhysicsComponent OtherBody(EntityUid uid)
+        {
+            if (uid == EntityA)
+                return BodyB!;
+            else if (uid == EntityB)
+                return BodyA!;
+
+            throw new InvalidOperationException();
+        }
```

Comment:
```suggestion

        [Pure, PublicAPI]
        public PhysicsComponent OurBody(EntityUid uid)
        {
            if (uid == EntityA)
                return BodyA!;
            else if (uid == EntityB)
                return BodyB!;

            throw new InvalidOperationException();
        }

        [Pure, PublicAPI]
        public PhysicsComponent OtherBody(EntityUid uid)
        {
            if (uid == EntityA)
                return BodyB!;
            else if (uid == EntityB)
                return BodyA!;

            throw new InvalidOperationException();
        }
```
Might as well add this one, too. All the other fields got both versions.